lib/sequence.js

```diff
@@ -216,15 +218,15 @@ module.exports = function SequenceFactory(connection) {
           return;
         }
         if (!_.isNull(createSeq)) {
-          doc[sequence._options.inc_field] = createSeq;
+          _.set(doc, sequence._options.inc_field.split('.'), createSeq);
```

Comment:
Document hs a `set` method which accept the path: https://mongoosejs.com/docs/api/document.html#document_Document-set

Maybe we don't need lodash for this. Also, the path taken can accept more complex structures as `"field.subfield[1].count"`

Reply:
Completely agree. But having used this lodash function before, it just came to my mind first than using the own document functions :sweat_smile: Thanks for the heads up.

I made a new commit with the code.